lib/nexpose/scan_template.rb

```diff
@@ -583,5 +583,20 @@ def enable_enhanced_logging=(enable)
       self.enable_debug_logging = enable
       self.aces_level = (enable ? 'full' : 'none')
     end
+
+    # Enable or disable windows service editor
+    # @param [Boolean] enable Enable or disable windows service editor
+    def windows_service_editor=(enable)
+      cifs_scanner = REXML::XPath.first(@xml, 'ScanTemplate/Plugins/Plugin[@name="java/CifsScanner"]')
+      param = REXML::XPath.first(cifs_scanner, './param[@name="windowsServiceEditor"]')
+      if param
+        param.text = (enable ? 1 : 0)
```

Comment:
1. Used in code I based on, but you're correct. I'll change it.
2. Not entirely
```
$ grep -n '1 : 0' lib/nexpose/scan_template.rb 
185:        param.text = (enable ? 1 : 0)
189:        param.text = (enable ? 1 : 0)
198:      icmp.attributes['enabled'] = (enable ? 1 : 0)
205:      tcp.attributes['enabled'] = (enable ? 1 : 0)
219:      udp.attributes['enabled'] = (enable ? 1 : 0)
577:      debug_logging.attributes['enabled'] = (enable ? 1 : 0)
593:        param.text = (enable ? 1 : 0)
597:        param.text = (enable ? 1 : 0)
```
The majority though use strings though. I'll do the same as suggested